### DISCUSSION TITLE
Pitch damping ratio limits

### DISCUSSION THREAD

**Discussion by ximenovoa** [NatLabRockies/ROSCO#335] [Q&A] [answered]
Hello all,

I have been running some simulations with ROSCO in a floating turbine IEA 15MW but wanted to ask a little bit more about the limits for the pitch controller damping (zeta_pc). In control is usual to use underdamped systems, where zeta<1, however, I have noticed that when increasing the damping to an overdamped system (zeta>1), the power increases and the loads decrease in my system, improving the results I get. In addition, I found some papers where, while using ROSCO and optimizing this parameter, the value of zeta_pc used goes to even more than 2 (like in [this paper](https://iopscience.iop.org/article/10.1088/1742-6596/2265/4/042021)). 

This caught my attention since in overdamped systems both of the poles are on the real axis and when the damping ratio is increased, one goes to infinity, and the other approaches to the origin. The dynamic of the pole going to infinity is so fast that can be neglected, but the dynamic of the pole going to the origin becomes slower and this dynamic is the one that will prevail. Therefore, applied to the pitch controller damping, does it make sense to use an overdamped system that in a real dynamic could make the angle change velocity slower? And if it makes sense, how is it possible to establish a range of damping values that physically make sense? 

Any help with this is really appreciated! Thanks.

**Comment by dzalkind** [comment, marked as answer]
Hi @ximenovoa, this was a common complaint from my PhD committee :-). However, it was also hard to argue with the improved performance of the overdamped system. 

I think all the values physically make sense because they map to PI gains.  From a classical second-order systems theory perspective, perhaps damping ratio is not the correct term, but it's still helpful for tuning pitch controllers. I like to think that in an highly overdamped system, there's a combined fast/slow response. I wrote a more thorough analysis is in this paper, Section 3.2.1: https://wes.copernicus.org/articles/5/1579/2020/ 

I hope this helps.  If you have further suggestions and thoughts, they are most welcome.

Best, Dan

**Reply by ximenovoa** [reply]
Hi Dan, with all that it makes more sense, thanks a lot for the explanation and the reference!

Best,
_Ximena_
